app/parse.py

```diff
@@ -1,9 +1,23 @@
+import csv
+import time
 from dataclasses import dataclass
 from urllib.parse import urljoin
 
+from selenium import webdriver
+from selenium.common import NoSuchElementException
+from selenium.webdriver.common.by import By
+from tqdm import tqdm
 
 BASE_URL = "https://webscraper.io/"
 HOME_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/")
+COMPUTERS_URL = urljoin(HOME_URL, "computers")
+LAPTOPS_URL = urljoin(HOME_URL, "computers/laptops")
+TABLETS_URL = urljoin(HOME_URL, "computers/tablets")
+PHONES_URL = urljoin(HOME_URL, "phones")
+TOUCH_URL = urljoin(HOME_URL, "phones/touch")
+
+options = webdriver.ChromeOptions()
+options.add_argument("--headless")
 
 
 @dataclass
```

Comment:
let's don't overwhelm this file. create a separate file for classes, dataclasses, constants, etc